packages/middleware-sdk-glacier/src/account-id-default.ts

```diff
@@ -11,7 +11,7 @@ export function accountIdDefaultMiddleware(): InitializeMiddleware<any, any> {
   return <Output extends MetadataBearer>(next: InitializeHandler<any, Output>): InitializeHandler<any, Output> =>
     async (args: InitializeHandlerArguments<any>): Promise<InitializeHandlerOutput<Output>> => {
       const { input } = args;
-      if (input.accountId === undefined) {
+      if (!input.accountId) {
```

Comment:
protocol tests say this should be inserted for both null and empty string, not only null.